edx/edx-app-ios
# Swipe To Delete Video Functionality

### Description

[LEARNER-1289] (https://openedx.atlassian.net/browse/LEARNER-1289)
Swipe to delete video functionality

### How to test this PR

- Go in Video Section
- Download all videos 
- Swipe the row from right to left, the delete button will appear.
- On Sub-section level delete action will delete all video components in subsection
- On Unit level delete action will delete just one video component in unit.
- Swipe to delete option will be enable when all videos are downloaded or the downloaded icon is check sign.
